### PR TITLE
Kevin/new mode

### PR DESCRIPTION
**Pull request by RocketKevin**
-Medium Mode
-Select box
-Remove easy and hard button

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -146,12 +146,32 @@ function easy() {
...
   var mode = "hard";
   localStorage.setItem("mode", mode);
   window.location.reload();
 }
 
+function generate() {
+  if(document.readyState === "complete") {
+    let selectMenu = document.getElementById("selectMenu");
+    let modeVal = selectMenu.options[selectMenu.selectedIndex].value;
+    if(parseInt(modeVal) === 1) {
```
Comment (brotatotes) [inline src/App.js:165]: make sure it's not 0, 1, 2

File: src/App.js [hunk 2]
```diff
@@ -395,6 +415,8 @@ class Board extends React.Component {
     let amountRemove = 46;
     if(mode === "easy") {
       amountRemove = 15;
+    } else if(mode === "medium") {
```
Comment (brotatotes) [inline src/App.js:418]: I think difficulty values should be setup as globals, maybe as enums?

File: src/App.js (hunk 1 above)
Comment (RocketKevin) [reply src/App.js]: Fixed

File: src/App.js (hunk 2 above)
Comment (RocketKevin) [reply src/App.js:424]: So, instead of mode, it's difficulty values be global?

Comment (RocketKevin) [reply src/App.js]: done

**Pull request merged**